backend/graph/api/schema/schema.graphqls

```diff
@@ -106,6 +106,8 @@ type QueryRoot{
   profiles: [Profile!]!
   profile: Profile!
 
+  analytics: Analytics!
```

Comment:
Would it make sense to have this under `{ me { analytics {...} } }`? Feels like user related data